assets/externalized/game.json

```diff
@@ -117,6 +117,10 @@
 	//  Always show mouse cursor during tactical view (if false, no mourse cursor is shown when moving in real-time mode, selecting a merc, etc)
 	//  vanilla setting: false
 	"always_show_cursor_in_tactical": false,
+	
+	//  Show chance-to-hit when pressing 'F' and next to mouse cursor when preparing an attack
+	//  vanilla setting: false
+	"show_hit_chance": true,
```

Comment:
If we're adding a new config key, then I'm for removing the hotkey. Or at least making it a toggle like caps lock.

Reply:
You mean removing 'F' hot key? That also shows range information, if I recall. I think that was the intended usage of the hot key.

Reply:
Just the extra functionality.

Reply:
I'm okay with that. I doubt anyone would want to only see hit chance by pressing 'F' when they can make it visible on the mouse cursor.